data/udev-filters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Disk" : {
+	"allowed-path-patterns" : [ "ccw-*", "scsi-*", "pci-*", "platform-*" ],
+	"allowed-id-patterns" : [ "ata-*", "scsi-*", "wwn-*",
+				  "nvme-uuid.*", "nvme-eui.*", "nvme-nvme.*",
+				  "usb-*", "mmc-*" ]
+    },
+    "Dasd" : {
+	"allowed-path-patterns" : [ "ccw-*" ],
+	"allowed-id-patterns" : [ "ccw-*" ]
+    },
+    "Md" : {
+	"allowed-id-patterns" : [ "md-uuid-*" ]
+    },
+    "DmRaid" : {
+	"allowed-id-patterns" : [ "raid-*" ]
+    },
+    "Multipath" : {
+	"allowed-id-patterns" : [ "scsi-*", "wwn-*", "dm-name-*", "dm-uuid-*" ]
```

Comment:
What exactly do you need these for? For multipath devices, we recommend 
`dm-uuid-mpath-*` and for partitions on multipath devices `dm-uuid-part*-mpath-*`. See also the [SLE storage guide, §18.2.4](https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-multipath.html#sec-multipath-referring).

If you need to use the lower-layer links (like `scsi-*`) here, you may also need to include the `ccw-*` and possible `nvme-*` links.